packages/apps/workshop/README.md

```diff
@@ -6,6 +6,7 @@ Install and run Storybook in a dev mode
 
 ```sh
 yarn
+yarn build:dependency
```

Comment:
Shouldn't this be integrated in `yarn:start` ? I'm not sure we should be worrying what to start and to build when starting storybook. For instance in `manager` repository we have the `start:dev` command that will handle the build of needed dependencies so we don't have to worry about it